src/react/client/client.js

```diff
@@ -1,26 +1,48 @@
 import { mockResponse } from './client.mock'
+import { invokeAlertDialog } from './alertDialog'
 
-const { ipcRenderer } = window
+let { ipcRenderer } = window
 
-export const openChannel = (channel, params) => {
-  if (!ipcRenderer) {
-    console.log('IPC client mock response', channel, params)
+if (process.env.NODE_ENV === 'test') {
+  ipcRenderer = require('electron').ipcRenderer
+  window.frontendLogger = { info: () => {}, error: () => {} }
+}
+
+if (process.env.NODE_ENV === 'development') {
+  ipcRenderer = undefined
+  window.frontendLogger = { info: console.log, error: console.error }
+}
+
+/**
+ * Sends an event to given channel with given parameters
+ *
+ * @param Channel name
+ * @param Parameters
+ *
+ * @return Promise -> resolves with responseObject, rejects on Error object
+ *
+ * @example invokeChannel(channels.GENERATE_DIOGRAPH, address)
+ *
+ */
+export const invokeChannel = (channel, params) => {
+  // Development uses mock responses
```

Comment:
Why development uses mock responses?

Reply:
Because localhost:3300  it doesn't have backend...